lib/output-jut.js

```diff
@@ -23,23 +23,29 @@ var JutOutput = module.exports = function JutOutput(opts)
 	this.target = opts.target;
 	this.request = opts.request || require('request');
 
-	this.logger = bole(this.toString());
+	this.log = bole('jut');
+	this.log.info('jut output configured');
 };
 util.inherits(JutOutput, stream.Writable);
 
-JutOutput.prototype.client = null; // client for data sink
-JutOutput.prototype.batch = [];
+JutOutput.prototype.client    = null; // client for data sink
+JutOutput.prototype.batch     = [];
 JutOutput.prototype.batchSize = 1000;
+JutOutput.prototype.lasterror = 0;
+JutOutput.prototype.THROTTLE  = 300000; // 5 minutes
+JutOutput.prototype.log = null;
 
 JutOutput.prototype._write = function _write(event, encoding, callback)
 {
-	var point = { source_type: 'metric', pid: process.pid };
-	_.assign(point, event);
-	if (point.tags) delete point.tags; // influx client does not like arrays
-	if (!point.time)
-		point.time = (new Date()).toISOString();
-	if (typeof point.time !== 'object')
-		point.time = new Date(point.time).toISOString();
+	if (event.name === 'heartbeat') return callback();
+	var point = _.pick(event, function(v) { return !_.isObject(v) && !_.isArray(v); });
+	if (point.metric && !point.value)
```

Comment:
Uh oh. I just realized that we emit both `metric` and `value`. I think we should standarize on `value` instead of doing this, what do you think?